code/code/misc/tweaks.h

```diff
@@ -0,0 +1,78 @@
+//////////////////////////////////////////////////////////////////////////
+//
+// SneezyMUD - All rights reserved, SneezyMUD Coding Team
+//
+//  tweaks.h
+//
+//
+//  A tweak is generally a coeffecient that gets slapped in the code 
+//  that can be modified in-game.  To add a new tweak 1) add the tweak 
+//  type to the enum tweakTypeT right above MAX_TWEAK_TYPES.  2)  Modify
+//  tweakInfoT::getTweakName to provide a name (no spaces).  3) Multiply 
+//  by tweaks[YOUR_TWEAK_TYPE]->cvalue somewhere in the code.  4)  The 
+//  tweak value defaults to 1.0.  Adjust the value with a trusted 
+//  immortal in game using info tweak.  If it is unnacceptable for the 
+//  value to start at 1.0, add a migration to migrations.cc inserting 
+//  the desired value.
+//
+//  Note that if you are changed the value of related tweaks at the same 
+//  time it might be hard to figure out the effect in game.
+//
+//////////////////////////////////////////////////////////////////////////
+
+
+#ifndef __TWEAK_H
+#define __TWEAK_H
```

Comment:
Every compiler under the sun understands `#pragma once`, which is 2 lines less boilerplate AND safer against user error